### DISCUSSION TITLE
FluentNumberField min and max value validation

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [microsoft/fluentui-blazor#350] [General]
Hi @vnbaaij,

I started writing test cases against FluentNumberField for increasing the coverage of components using `FluentInputBase<TValue>` and following up my previous PR (#341). I see that the min and max values are either a, provided or b, coming from the given type. What I realised is the following:

- the input is not validated, so in case of `int` I can provide higher value than `2147483647`
- I can provide higher value for the `min` than the `max` (I haven't tried what is happening here)

What is your take in this case? If you think this level of input validation is nice to have I'm happy to include them in my PR.

**Comment by vnbaaij** [comment]
Yes, would be good to have that kind of input validation. Please include!

**Reply by Andras-Csanyi** [reply]
Following this topic up, I'm working on it and 80-90% done. I'm changing position in my job and it requires more time than I have. End of next week I'll provide the PR.

**Reply by vnbaaij** [reply]
No rush!! Take your time.
